crates/fortitude_linter/src/rules/style/useless_return.rs

```diff
@@ -8,7 +8,7 @@ use ruff_source_file::SourceFile;
 use tree_sitter::Node;
 
 /// Get the next sibling that isn't a comment
-fn next_non_comment_sibling<'a>(node: &Node<'a>) -> Option<Node<'a>> {
+pub(crate) fn next_non_comment_sibling<'a>(node: &Node<'a>) -> Option<Node<'a>> {
```

Comment:
Maybe move this and `BlockExit` to `ast/types.rs`?

Reply:
Happy to do so. I've just approved https://github.com/PlasmaFAIR/fortitude/pull/559, so to avoid a messy merge conflict I'll wait until that's in and I've rebased on `main`.